picard/tagger.py

```diff
@@ -325,33 +325,41 @@ def pipe_server(self):
             if messages:
                 self.load_to_picard(messages)
 
-    def load_to_picard(self, items):
-        files = set()
-        mbids = set()
-        urls = set()
+    @staticmethod
+    def _parse_file_or_url_items(items):
+        out = {
+            "files": set(),
+            "mbids": set(),
+            "urls": set(),
+        }
```

Comment:
Alternatively, we could use a class, see https://github.com/skelly37/picard/pull/3